src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
```

Comment:
It seems like this might be cleaner, especially if you pass in the sb instances to `ScanUnicodeEscapeSequence`.